src/main/java/dev/sigstore/rekor/client/RekorEntry.java

```diff
@@ -15,13 +15,21 @@
  */
 package dev.sigstore.rekor.client;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Base64.getDecoder;
+
+import dev.sigstore.json.GsonSupplier;
+import dev.sigstore.rekor.Hashedrekord;
 import org.immutables.gson.Gson;
 import org.immutables.value.Value;
 
 /** A local representation of a rekor entry in the log. */
 @Gson.TypeAdapters
 @Value.Immutable
 public interface RekorEntry {
+
+  com.google.gson.Gson GSON = new GsonSupplier().get();
```

Comment:
`GSON` -> `gson`
I think you can clean up the fully qualified name and just import it?

Reply:
oh it's a const... nvm?

Reply:
ha, I hadn't even realized I'd introduced a field on an interface. Now I have to read-up on how that works... for now I'm going to in-line, it doesn't make sense as a public part of the interface.  Maybe we can think about making it a singleton later?